bitcoin/src/blockdata/script/owned.rs

```diff
@@ -11,6 +11,9 @@ use crate::opcodes::all::*;
 use crate::opcodes::{self, Opcode};
 use crate::prelude::{Box, Vec};
 
+#[cfg(feature = "arbitrary")]
+use arbitrary::{Arbitrary, Result, Unstructured};
```

Comment:
This seems suspicious, why would we want to override `std::result::Result` when `arbitrary` is enabled?

Reply:
Definitely should prefer `arbitrary::Result` to avoid conflicts. Or put the whole thing in a separate module.

Reply:
I'm perplexed by this comment.  We are using `arbitrary::Result` when arbitrary is enabled.  This is copy pasta from the exmaple in the readme: https://github.com/rust-fuzz/arbitrary under the section `implementing Arbitrary By Hand`.  I feel like I'm missing the point of this comment.

Reply:
@tcharding @Kixunil I'd like to resolve this PR although I don't understand what you two are driving at here.  Kindly asking for a better explanation so I can resolved this.

Reply:
Simply don't import `Result` and instead write `arbitrary::Result` instead of `Result`.

The reason is if we need to add something returning normal `Result` we will have annoying name conflicts.

Reply:
> The reason is if we need to add something returning normal Result we will have annoying name conflicts.

well only if cfg arbitrary is being used.  Sounds good though, I've updated it so this should be resolved.

Reply:
thanks for the prompt response

Reply:
(FTR in my original review I framed it as a question so in answering it you could see why it was wrong instead of just being that grumpy old bastard who says "don't do that".)